flair/models/text_classification_model.py

```diff
@@ -478,7 +478,7 @@ def _fetch_model(model_name) -> str:
         
         #Communicative Functions Model
         model_map["communicative-functions"] = "/".join(
-            [hu_path, "comfunc", "communicative-functions-v0.5b.pt"]
+            [hu_path, "comfunc", "communicative-functions.pt"]
```

Comment:
This should be the new model, right?

Reply:
The new model is called "communicative-function.pt" without the end part "-v0.5b". I admit its confusing.

Reply:
This change is indeed still required for the new model to be downloaded.